scripts/convert_metadata.py

```diff
@@ -0,0 +1,361 @@
+                        # For each dimension component, find the standard_name
+                        # First, see if it is blank, then integer, then var
+                        fd = fdlist[index]
+                        if len(fd) == 0:
+                            fdnum = ''
+                        else:
+                            fdnum = None
+                        # End if
+                        if len(fd) > 0:
+                            try:
+                                test = int(fd)
+                                fdnum = fd
+                            except Exception as e:
+                                pass
+                            # End try
+                        # End if
+                        if (len(fd) > 0) and (fdnum is None):
+                            # Try to find the standard name for this item
+                            if table.has(fd):
+                                fdvar = table.get(fd)
+                                fdnum = fdvar['standard_name']
+                            else:
+                                print('WARNING: No local variable found for dimension, {} in {}'.format(fd, lname))
+                                fdnum = 'XXnot_foundXX'
+                            # End if
+                        # End if
+                        dlist[index] = fdnum
+                    # End for
+                    # Reassemble dim
+                    dims.append(':'.join(dlist))
+                # End for
+                # Reset dimensions
+                var['dimensions'] = '(' + ','.join(dims) + ')'
+            # End if
+        # End for
+    # End for
+    # Write out finalized metadata file
+    with open(metadata_filename_out, 'w') as mdfile:
+        spacer = ""
+        for table in mdconfig:
+            if len(spacer) > 0:
+                mdfile.write(spacer)
+            # End if
+            table.write(mdfile)
+            spacer = '\n'+72*'#'+'\n'
+        # End for
+    # End with (mdfile)
+
+########################################################################
+
+def usage(cmd):
+    print("Usage:")
+    print("{} <source_file> <target_file>".format(cmd))
+    print("{} <source_file> [ <source_file> ...] [ <target_directory>".format(cmd))
+    print("")
+    print("Translate the metadata in each <source_file> into a new file")
+
+########################################################################
+
+if __name__ == "__main__":
+    # Process the files passed in
+    num_args = len(sys.argv)
+    if num_args < 3:
+        usage(sys.argv[0])
+    else:
+        if os.path.isdir(sys.argv[-1]):
+            target_dir = os.path.abspath(sys.argv[-1])
+            num_args = num_args - 1
+            for index in xrange(1, num_args):
+                source_file = os.path.abspath(sys.argv[index])
+                filename = os.path.basename(source_file)
+                mdfilename = "{}.md".format('.'.join(filename.split('.')[:-1]))
```

Comment:
A potential downside to using a file suffix of `.md` is that this is already used by GitHub to identify markdown files. Should we consider an alternative suffix like `.meta`?

Reply:
I agree with the concern and with the `.meta` name suggestion.

Reply:
@climbfuji, @grantfirl, @llpcarson, can you weigh in on this issue?

Reply:
I had the same concern about .md too, so I'm OK with .meta. Looks like the only conflict with .meta has to do with them being used for something in RealPlayer, so I can't imagine any confusion there.